src/libs/actions/Report.js

```diff
@@ -158,6 +158,97 @@ function getSimplifiedReportObject(report) {
     };
 }
 
+/**
+ * Get a simplified version of an IOU report
+ *
+ * @param {Object} reportData
+ * @param {Number} reportData.transactionID
+ * @param {Number} reportData.amount
+ * @param {String} reportData.currency
+ * @param {String} reportData.created
+ * @param {String} reportData.comment
+ * @param {Object[]} reportData.transactionList
+ * @param {String} reportData.ownerEmail
+ * @param {String} reportData.managerEmail
+ * @param {Number} reportData.reportID
+ * @returns {Object}
+ */
+function getSimplifiedIOUReport(reportData) {
```

Comment:
Why do we need to do this? Why not save the full reportData in onyx?

Reply:
From https://github.com/Expensify/Expensify/issues/157104, those are the only fields we wanted to store in the IOU collection. Do we want to store more of them? cc @Julesssss

Reply:
II don't see why we need too. If we do store the full report then we may as well merge the Onyx collection with the reports collection as previously discussed.

Reply:
True, but I am unsure why we did that, why transform the object into a slightly different one?

Reply:
Simple object == simple to understand was the idea, also to help reduce Onyx storage, which we believe will eventually be a limitation.

Having said that, I am not against reverting to the full object.

Reply:
I'd assume we'd only want to store the data we are going to use for the other IOU components. That's the reason we're using `getSimplifiedReportObject`:
```
* Only store the minimal amount of data in Onyx that needs to be stored
 * because space is limited
```